compiler/luci/pass/src/FuseBCQPass.cpp

```diff
@@ -38,6 +38,16 @@ const std::string node_name_prefix(luci::NodeName node_name)
 {
   std::string prefix = node_name;
 
+  if (prefix.find("ReadVariableOp/resource/") != std::string::npos)
+  {
+    const auto start_index = prefix.find("ReadVariableOp/resource/");
```

Comment:
As `"ReadVariableOp/resource/"` appears twice, how about adding a variable const std::string or const char * for this?